### ISSUE TITLE
arm9loaderhax.bin: Failed to load key X for slot 0x | Code: 4

### ISSUE THREAD

**Issue by joedude1635** [roxas75/rxTools#448]
I get this when I use the arm9loaderhax.bin payload. The installation suite fails, and then it shows this. I have the correct firm folder.

**Comment by Desterly** [comment]
It's looking for the slot0x25KeyX.bin file because it thinks firmware is < 7.

@AlbertoSONIC couple items here.  First the text is getting cut off so it ends at "slot 0x"
second, I'm assuming that we need to remove this check for arm9loader or identify sysver differently?

**Comment by urherenow** [comment]
If keys are needed, there is an issue anyway because a9lh works before keys (or any firm at all) are even initiated...

**Comment by joedude1635** [comment]
So is this a bug for everyone? My sysNAND is on 9.2.

**Comment by AlbertoSONIC** [comment]
Yeah I noticed this too. A workaround is, obviously, to put slot0x25keyx.bin in the sd even if not needed. I'll disable that check when booted from arm9loaderhax later ;)

**Comment by joedude1635** [comment]
The installation is still failing. Then, when I try and load sysNAND or emuNAND, it says "Failed to load NATIVE_FIRM: 4"

**Comment by joedude1635** [comment]
Anyone have a solution? I've tried booting it by itself and with CTRBootManager. Here is a picture of the root of my SD card and the rxTools directory: http://imgur.com/a/6ZJpN

**Comment by urherenow** [comment]
Use AUReinand to launch sysnand with the 9.0 firm, then use browserhax or hblauncher_loader to launch HBL (or ctrbootmanager). Launching rxTools from there will complete the data installation suite, then you can replace arm9loaderhax.bin with rxTools'.

**Comment by joedude1635** [comment]
Hmm... seem to get into sysNAND... I didn't uninstall menuhax before I installed arm9loaderhax, so that might be it. My rei folder has an emunand folder, a reboot folder, a firmware90.bin, and an updatedsysnad file (with no extension).

**Comment by mariogamer2** [comment]
Personally, my o3ds do not boot at all...
